packages/block-library/src/cover/shared.js

```diff
@@ -21,11 +21,16 @@ export const VIDEO_BACKGROUND_TYPE = 'video';
 export const COVER_MIN_HEIGHT = 50;
 export const COVER_MAX_HEIGHT = 1000;
 export const COVER_DEFAULT_HEIGHT = 300;
+export const DEFAULT_FOCAL_POINT = { x: 0.5, y: 0.5 };
 export function backgroundImageStyles( url ) {
 	return url ? { backgroundImage: `url(${ url })` } : {};
 }
```

Comment:
Is `backgroundImageStyles` still being used anywhere?

We may be able to move it to the deprecations file if not.

Reply:
Deprecated here: https://github.com/WordPress/gutenberg/pull/40554/commits/6aa0b4870991baa2633bdf71826711c240587f1e